<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Inci-MacBook-Air.local" failures="4" tests="7" name="All links test" time="12.204" errors="0" timestamp="2020-07-03T02:14:24 CDT">
  <testcase classname="Main.java.testNGclasses.A14__TNG_Listener1" name="contributeTest" time="2.263"/>
  <testcase classname="Main.java.testNGclasses.A14__TNG_Listener1" name="contributeTest" time="2.114"/>
  <testcase classname="Main.java.testNGclasses.A14__TNG_Listener1" name="contributeTest" time="1.651"/>
  <testcase classname="Main.java.testNGclasses.A14__TNG_Listener1" name="readData" time="0.005">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Main.java.testNGclasses.A14__TNG_Listener1.readData(A14__TNG_Listener1.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- readData -->
  <testcase classname="Main.java.testNGclasses.A14__TNG_Listener1" name="readData" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Main.java.testNGclasses.A14__TNG_Listener1.readData(A14__TNG_Listener1.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- readData -->
  <testcase classname="Main.java.testNGclasses.A14__TNG_Listener1" name="readData" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Main.java.testNGclasses.A14__TNG_Listener1.readData(A14__TNG_Listener1.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- readData -->
  <testcase classname="Main.java.testNGclasses.A14__TNG_Listener1" name="timeOutTest" time="1.013">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method Main.java.testNGclasses.A14__TNG_Listener1.timeOutTest() didn&amp;apos;t finish within the time-out 1000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method Main.java.testNGclasses.A14__TNG_Listener1.timeOutTest() didn't finish within the time-out 1000
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- timeOutTest -->
  <testcase classname="Main.java.testNGclasses.A14__TNG_Listener1" name="test2" time="0.0">
    <ignored/>
  </testcase> <!-- test2 -->
</testsuite> <!-- All links test -->
